deploy/example_catalog/docs/mapr610-readme.md

```diff
@@ -1,5 +1,5 @@
 ## Overview
-The app is built with MapR 6.1 MEP 6.3 components. There are 2 set of kubedirector apps, the difference between the 2 is related to security setting only.
+The app is built with MapR 6.1 MEP 6.3 components. This is only secured version of the Ecosystem
```

Comment:
The second sentence here is kind of cryptic. Is it necessary?

Reply:
yeah removed it